app/forms/state_file/sales_use_tax_form.rb

```diff
@@ -0,0 +1,35 @@
+module StateFile
+  class SalesUseTaxForm < QuestionsForm
+    set_attributes_for :intake,
+                       :untaxed_out_of_state_purchases,
+                       :sales_use_tax_calculation_method,
+                       :sales_use_tax
+
+    validates :untaxed_out_of_state_purchases, presence: true
+    validates :sales_use_tax_calculation_method, presence: true, if: -> { untaxed_out_of_state_purchases == "yes" }
+    validates :sales_use_tax,
+      presence: true,
+      numericality: {
+        message: ->(_object, _data) { I18n.t("state_file.questions.sales_use_tax.edit.enter_valid_dollar_amount") },
```

Comment:
I'd bet this is the bit that's failing for the 'unused' i18n bit. If it is, the `activerecord.errors.models` [bit of the locale files](https://github.com/codeforamerica/vita-min/blob/ec24141cbeb4da680bbdfadc6490dd6984c93aee/config/locales/en.yml#L12) is for this sort of things, but with us using forms rather than models I don't imagine it would get picked up. But maybe it would since it is handled in an ancestor class by including activemodel? Could be worth a quick smoke test to see if it gets picked up.

Reply:
Whoops, noticed this is wip lol. Need to update my filter, sorry